src/js/brutusin-json-forms.js

```diff
@@ -177,6 +177,7 @@ if (typeof brutusin === "undefined") {
             } else if (s.media) {
                 input = document.createElement("input");
                 input.type = "file";
+                appendChild(input, option, s);

```

Comment:
Addition of this line is unintended. I will remove this line from my version and resubmit the pull request.